flang/include/flang/Frontend/LangOptions.h

```diff
@@ -58,6 +61,9 @@ class LangOptions : public LangOptionsBase {
   /// host code generation.
   std::string OMPHostIRFile;
 
+  /// List of triples passed in using -fopenmp-targets.
+  std::vector<llvm::Triple> OMPTargetTriples;
```

Comment:
`std::vector` is fine, but did you consider `SmallVector<T,0>`?

> [SmallVector has grown a few other minor advantages over std::vector, causing SmallVector<Type, 0> to be preferred over std::vector<Type>.](https://llvm.org/docs/ProgrammersManual.html#llvm-adt-smallvector-h)

Reply:
Yes, I decided to use `std::vector` just because it seems to be the preferred choice for arrays in the different `Fortran::frontend::*Options` classes.